apps/frontend/src/shared-research/__tests__/ResearchOutput.test.tsx

```diff
@@ -68,31 +69,54 @@ const renderComponent = async () => {
   return result;
 };
 
-it('renders the research output', async () => {
-  mockGetResearchOutput.mockResolvedValue({
-    ...createResearchOutputResponse(),
-    title: 'Proposal title!',
+describe('proposal research output', () => {
+  it('renders a research proposal', async () => {
+    mockGetResearchOutput.mockResolvedValue({
+      ...createResearchOutputResponse(),
+      type: 'Proposal',
+      team: {
+        displayName: 'Proposal Team',
+        id: '123',
+      },
+      title: 'Proposal title!',
+    });
+    const { getByRole } = await renderComponent();
+    expect(getByRole('heading').textContent).toEqual('Proposal title!');
   });
-  const { getByRole } = await renderComponent();
-  expect(getByRole('heading').textContent).toEqual('Proposal title!');
-});
+  it('renders the research output with a team', async () => {
+    mockGetResearchOutput.mockResolvedValue({
+      ...createResearchOutputResponse(),
+      type: 'Proposal',
+      team: {
+        id: '0d074988-60c3-41e4-9f3a-e40cc65e5f4a',
+        displayName: 'Sulzer, D',
+      },
+    });
 
-it('renders the research output with a team', async () => {
-  mockGetResearchOutput.mockResolvedValue({
-    ...createResearchOutputResponse(),
-    team: {
-      id: '0d074988-60c3-41e4-9f3a-e40cc65e5f4a',
-      displayName: 'Sulzer, D',
-    },
+    const { getByText } = await renderComponent();
+    const element = getByText('Team Sulzer, D');
+    expect(element).toBeVisible();
+    expect(element).toHaveAttribute(
+      'href',
+      '/network/teams/0d074988-60c3-41e4-9f3a-e40cc65e5f4a',
+    );
   });
+});
 
-  const { getByText } = await renderComponent();
-  const element = getByText('Team Sulzer, D');
-  expect(element).toBeVisible();
-  expect(element).toHaveAttribute(
-    'href',
-    '/network/teams/0d074988-60c3-41e4-9f3a-e40cc65e5f4a',
-  );
+describe('other research outputs', () => {
+  it('renders a research output', async () => {
+    mockGetResearchOutput.mockResolvedValue({
+      ...createResearchOutputResponse(),
+      type: 'Protocol',
+      tags: ['Example Tag'],
+      title: 'Proposal title!',
+    });
+    const { getByRole, getByText } = await renderComponent();
+    expect(getByText(/Example Tag/i)).toBeVisible();
+    expect(getByRole('heading', { level: 1 }).textContent).toEqual(
+      'Proposal title!',
+    );
+  });
 });
```

Comment:
The describe and it titles are a bit of a mess with the new split, to suggest a format maybe something like
describe 'a non-proposal research output'
  it 'is rendered with tags'